### ISSUE TITLE
Bug Report: TypeError when Removing Validators

### ISSUE THREAD

**Issue by Kani999** [TheDJVG/netbox-field-permissions#3]
### Steps to Reproduce:
1. Visit the following URL: `INSTANCE_URL/plugins/netbox_field_permissions/field-permissions/manage/`
2. Ensure validators are installed (if not already installed).
3. Click on `Remove Validators`.
    - POST on https://INSTANCE_URL/plugins/netbox_field_permissions/field-permissions/manage/
    - /netbox_field_permissions/utilities.py, line 120, in uninstall_validator

### Expected Behavior:
Validators should be removed without error.

### Actual Behavior:
A `TypeError` occurs when attempting to remove validators. `list indices must be integers or slices, not str`

### Log output
```
TypeError at /plugins/netbox_field_permissions/field-permissions/manage/

list indices must be integers or slices, not str

Request Method: | POST

https://INSTANCE_URL/plugins/netbox_field_permissions/field-permissions/manage/
5.0.9
TypeError
list indices must be integers or slices, not str
/opt/netbox/venv/lib/python3.12/site-packages/netbox_field_permissions/utilities.py, line 120, in uninstall_validator
netbox_field_permissions.views.FieldPermissionManageView
/opt/netbox/venv/bin/python
3.12.3
```

**Comment by Kani999** [comment]
We are encountering a problem related to **Custom Validation** in NetBox, which is implemented as per the [NetBox documentation on custom validation](https://netboxlabs.com/docs/netbox/en/stable/customization/custom-validation/). The validation is set up using the `configuration.py` file.

However, this `Custom Validation` breaks the logic for the `netbox-field-validator`, which uses `Config Revision` to set up the validators.

Maybe someone will find this information useful.

**Comment by Kani999** [comment]
I see a message `Static NetBox configuration detected. Please install customer validator manually`

```python
def write_static_configuration_messsage():
    sys.stdout.write(
        colorize(
            "\n  Static NetBox configuration detected. Please install customer validator manually!!",
            fg="yellow",
            opts=("bold",),
        )
    )
    sys.stdout.flush()
```

___ 
Can you please tell me, how can I install my own custom validator manually? 

It means, I cannot add them in `configuration.py` and need to add them "manually" to the config revision? 
I did not found any other hint to how to do that?

**Comment by TheDJVG** [comment]
Hi @Kani999,

Thanks for reporting this. The migrations do indeed check if NetBox uses a static validator configuration and prevents activating a dynamic configuration. 

The original plan did include a view to help with install/uninstall but did not make it to the current release candidate. I'll will create a fix for this and the management page will show the resulting JSON that can be used to statically configure the validators. If you can't wait for that you can add `netbox_field_permissions.validators.FieldPermissionValidator` for each model in the `CUSTOM_VALIDATORS` dict.

**Comment by TheDJVG** [comment]
This is now fixed in release 0.1.0rc2
